### PR TITLE
Fix for travis ci.

### LINKED ISSUE #12: Tests are broken in TravisCI
On TravisCI the tests are run against recent versions of the dependencies. This currently fails: https://travis-ci.org/plone/plone.memoize/builds/418577118

The tests should be fixed because this package is also used outside Plone.

### PR DESCRIPTION
**Pull request by andbag**
Fixes #12

### CODE REVIEW

File: buildout.cfg [hunk 1]
```diff
@@ -9,7 +9,7 @@ python-gettext = >= 3
 zc.recipe.testrunner = >= 2
 zope.annotation = >= 4
 zope.browser = >= 2
-zope.component = >= 4
+zope.component = <= 4.2.1 , >= 4
```
Comment (icemac) [inline buildout.cfg]: It seems to do the trick, cool!
If this is a hard dependency for the code it should also be in `setup.py`. Additionally a new issue for fixing this dependency would be nice. zope.component==4.2.1 is nearly 5 years old!

**Comment by icemac** [comment]
The broken test on "Plone Jenkins CI - pull-request-5.2-3.7" seems to be unrelated.

**Comment by andbag** [comment]
I have adjusted the implicit dependencies of zope.component so that only current libraries are used. Currently all tests work except the "Changelog verifier".

**Comment by andbag** [comment]
@icemac sorry for the force pushes, but there are only minimal changes to review.

Now, all checks were successful and the PR is ready for merge.

File: CHANGES.rst [hunk 2]
```diff
@@ -17,6 +17,16 @@ Bug fixes:
 - *add item here*
 
 
+2.0.1 (16.01.2019)
```
Comment (icemac) [inline CHANGES.rst]: I do not see this release on PyPI, did I miss something?

File: CHANGES.rst [hunk 3]
```diff
@@ -17,6 +17,16 @@ Bug fixes:
...
+------------------
+
+Bug fixes:
+
+- Fix broken tests in TravisCI
+  https://github.com/plone/plone.memoize/issues/12
+  [andbag]
+
+  
 2.0 (15.01.2019)
```
Comment (icemac) [inline CHANGES.rst]: Even this one does not seem to be there.

Comment (andbag) [reply CHANGES.rst]: Unfortunately I have no idea how the workflow from github to pypi works. Who is responsible for this step? At the moment the master branch is on version 2.0 and I only counted the version up. I can also add PR #14 to version 2.0.

@icemac What should I do?

**Comment by andbag** [comment]
@icemac Since the project now uses the `towncrier` tool for change log management, I was able to fix the problem.

**Pull request merged**